src/main/java/core/basesyntax/service/impl/FileReaderServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileReaderService;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderServiceImpl implements FileReaderService {
+    @Override
+    public List<String> readFromFile(String fileName) {
+        List<String> fileEntries = new ArrayList<>();
+        File file = new File(fileName);
+        try (BufferedReader br = new BufferedReader(new FileReader(file))) {
+            String line;
+            while ((line = br.readLine()) != null) {
+                fileEntries.add(line);
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read the file " + fileName, e);
+        }
+        fileEntries.remove(0);

```

Comment:
this logic does not apply to this method. The logic of this method is only reading data from a file. Move it to the appropriate one